README.md

```diff
@@ -44,8 +45,8 @@ Fluent plugin to send messages to IRC server
 |time_key|key name for time|time|
 |time_format|time format. This will be formatted with Time#strftime.|%Y/%m/%d %H:%M:%S|
 |tag_key|key name for tag|tag|
-|command|irc command. `priv_msg` or `notice`|priv_msg|
-|channel_keys|keys used to format channel. %s will be replaced with value specified by channel_keys if this option is used|nil|
+|command|irc command. `privmsg` or `notice`|privmsg|
+|command_keys|keys used to format command. %s will be replaced with value specified by command_keys if this option is used|nil|
```

Comment:
First, I implemented as `command_key` option, but I felt `command_keys` is better in terms of `symmetry` with other options. Please let me know if you prefer `command_key`.